.weblate.json

```diff
@@ -1806,6 +1806,1087 @@
+            {
+                "name": "l10n_ke",
+                "filemask": "addons/l10n_ke/i18n/*.po",
+                "new_base": "addons/l10n_ke/i18n/l10n_ke.pot",
+                "language_regex": "^(sw)$"
+            },
+            {
+                "name": "l10n_ke_edi_tremol",
+                "filemask": "addons/l10n_ke_edi_tremol/i18n/*.po",
+                "new_base": "addons/l10n_ke_edi_tremol/i18n/l10n_ke_edi_tremol.pot",
+                "language_regex": "^(sw)$"
+            },
+            {
+                "name": "l10n_kh",
+                "filemask": "addons/l10n_kh/i18n/*.po",
+                "new_base": "addons/l10n_kh/i18n/l10n_kh.pot",
+                "language_regex": "^(km)$"
+            },
+            {
+                "name": "l10n_km",
+                "filemask": "addons/l10n_km/i18n/*.po",
+                "new_base": "addons/l10n_km/i18n/l10n_km.pot",
+                "language_regex": "^(ar|fr)$"
+            },
+            {
+                "name": "l10n_kr",
+                "filemask": "addons/l10n_kr/i18n/*.po",
+                "new_base": "addons/l10n_kr/i18n/l10n_kr.pot",
+                "language_regex": "^(ko)$"
+            },
+            {
+                "name": "l10n_kw",
+                "filemask": "addons/l10n_kw/i18n/*.po",
+                "new_base": "addons/l10n_kw/i18n/l10n_kw.pot",
+                "language_regex": "^(ar)$"
+            },
+            {
+                "name": "l10n_kz",
+                "filemask": "addons/l10n_kz/i18n/*.po",
+                "new_base": "addons/l10n_kz/i18n/l10n_kz.pot",
+                "language_regex": "^(ru)$"
+            },
+            {
+                "name": "l10n_latam_base",
+                "filemask": "addons/l10n_latam_base/i18n/*.po",
+                "new_base": "addons/l10n_latam_base/i18n/l10n_latam_base.pot",
+                "language_regex": "^(es_419|pt_BR)$"
+            },
+            {
+                "name": "l10n_latam_check",
+                "filemask": "addons/l10n_latam_check/i18n/*.po",
+                "new_base": "addons/l10n_latam_check/i18n/l10n_latam_check.pot",
+                "language_regex": "^(es_419|pt_BR)$"
+            },
+            {
+                "name": "l10n_latam_invoice_document",
+                "filemask": "addons/l10n_latam_invoice_document/i18n/*.po",
+                "new_base": "addons/l10n_latam_invoice_document/i18n/l10n_latam_invoice_document.pot",
+                "language_regex": "^(es_419|pt_BR)$"
+            },
+            {
+                "name": "l10n_lb_account",
+                "filemask": "addons/l10n_lb_account/i18n/*.po",
+                "new_base": "addons/l10n_lb_account/i18n/l10n_lb_account.pot",
+                "language_regex": "^(ar|fr)$"
+            },
+            {
+                "name": "l10n_lk",
+                "filemask": "addons/l10n_lk/i18n/*.po",
+                "new_base": "addons/l10n_lk/i18n/l10n_lk.pot"
+            },
```

Comment:
Not sure whether this is intended, but this entry is missing the `language_regex`. As a result, `l10n_lk` is being offered to be translated in all languages.